tests/state/state.go

```diff
@@ -163,7 +162,7 @@ type NodeState struct {
 	// indicates if the node is Closed.
 	Closed bool
 	// AddrInfo contains the peer information for the node.
-	AddrInfo peer.AddrInfo
+	AddrInfo client.PeerInfo
```

Comment:
suggestion: Why not also rename the variable.

```suggestion
	PeerInfo client.PeerInfo
```

Reply:
actually no longer needed. Removed it.